.env.template

```diff
@@ -7,3 +7,9 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+TRELLO_API_KEY=<FILL ME IN!!>
+
+TRELLO_API_TOKEN=<FILL ME IN !!!>
+
+TRELLO_TODO_LIST_ID=<FILL ME IN !!!>
```

Comment:
I think we're missing a `TRELLO_DONE_LIST_ID` variable here, and as suggested below we'd benefit from `TRELLO_BOARD_ID` too